packages/calcite-components/src/components/action/action.e2e.ts

```diff
@@ -123,6 +123,44 @@ describe("calcite-action", () => {
     ]);
   });
 
+  describe("aria property", () => {
+    it("should set aria properties on internal button element", async () => {
+      const page = await newE2EPage();
+      await page.setContent(html`<calcite-action></calcite-action>`);
+
+      const buttonSelector = `calcite-action >>> .${CSS.button}`;
+      const action = await page.find("calcite-action");
+      const button = await page.find(buttonSelector);
+
+      expect(await button.getProperty("ariaExpanded")).toBe(null);
+      expect(await button.getProperty("ariaHasPopup")).toBe(null);
+      expect(await button.getProperty("ariaPressed")).toBe(null);
+      // expect((await page.$eval(buttonSelector, (el) => el.ariaControlsElements ?? [])).length).toBe(0);
```

Comment:
If these are no longer relevant, let's remove them. Otherwise, can you create a follow-up issue to restore these assertions?

Reply:
Yes, these seemed to fail in our browser testing since the browser doesn't seem to have `ariaControlsElements` or other newer properties yet.